Automattic/republication-tracker-tool
# Set up automated releases

### Changes proposed in this Pull Request:



Almost the same setup as with our other plugins, with two notable changes:

1. Scripts that handle WPORG deployment and post-release tasks now live in https://github.com/Automattic/newspack-scripts
2. There is one additional step for the post-release step - synchronising `master` branch with the `release` one, so that release-generated changes (changelog, version numbers in files) are on `master`

The changes are pretty hard to test locally or before merge, here's what's expected after these changes are on `master` and `release` is updated afterwards:   
1. A new version tagged and released on Github* 
2. A new version on WPORG
3. Latest `CHANGELOG.md` and version updated in relevant files on `master`

\* There are no release-triggering changes here, but I've forced it by adding an empty release-triggering commit

Closes #94.

### How to test the changes in this Pull Request:

1. Run `npm run release:archive` and upload the created plugin ZIP to a new site
2. Verify the plugin is working there by placing the RTT widget in a widget area and visitng and article. You should be able to open the RTT modal and copy article content.
